content/events/launchpad-summit-ethcc.md

```diff
@@ -0,0 +1,37 @@
+---
+ eventName: "Launchpad Summit Paris" #the name of your track or event and its mandatory
+
+ eventDescription: "Launchpad is a blend of two key components:  a dynamic four-week virtual learning cohort, where residents actively participate in remote learning seminars, and an unforgettable one-week in-person "colo" Summit.  These summits comprise interactive presentations, technology deep dives, and unparalleled networking opportunities." #short description of your track or event limiting to 100-150 characters
+
+ #topics will show up as labels on the event card.Only a maximum of 4 tags will be displayed on the event card. Some referneces for topics - Blockchain, Web3, Cryptocurrency, Tech Taks,Workshop etc.
+
+ eventTopic: 
+   - "Web3"
+   - "Tech Talks"
+
+ eventType: "Conference" #Type of the event: Please choose one among the below options or just leave it blank
+
+ location: "Paris, France" or "Location TBD" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+ venueName: "Montparnasse - Exact location revealed upon registration!" #the event venue name (will show up on the event card) or just leave it blank
+
+ website: "[your_website_url](https://protocol.ai/blog/launchpad-summit-paris-2023/)" #make sure to have all the relevant information: dates, venue, program, ticketing (if any), etc. or just leave it blank
```

Comment:
@laeccles I just updated this event as per Ryo's inputs, where the website was different. Can you please check again to confirm if all data here are the latest and are good to be updated over the existing data (including the dates as well since they differ as well)